fourigui/_version.py

```diff
@@ -131,10 +131,7 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
             root = os.path.dirname(root)  # up a level
```

Comment:
please check, do we need this file?  I htink we don't in our new workflow.

Reply:
I figured this would be removed in a separate PR for deleting unused files.